### DISCUSSION TITLE
I have added Crumbling Platforms

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#67] [Q&A]
So I checked in the to-do list for sonic Worlds next and one of them said Crumbling Platforms and I added them to the game it was extremely easy

**Comment by Renhoex** [comment]
The To do list is outdated and needs to be organized, I've already added the crumbling platforms in a previous update
![GIF 20-08-2023 4-27-28 PM](https://github.com/Techokami/SonicWorldsNext/assets/26295900/68ab801b-dada-4328-ba1d-ce2cd3719e86)

Do you have an actual question related to the crumbling platforms?

And if you're going to continue to post in here make sure the topic is actually a question, you haven't shown any example of what you mean when you say you implemented crumbling platforms and as far as I can see you haven't asked a question. Are you asking about contributing to the project or about the implementation of platforms?
